dev-requirements-py3.txt

```diff
@@ -20,4 +20,6 @@ circleci==1.2.2
 pylint==2.6.0
 coloredlogs==14.0
 pandas==1.1.0
-pyarrow==1.0.0
\ No newline at end of file
+pyarrow==1.0.0
+prettytable==1.0.1
```

Comment:
why are those needed? we have them in the build reqs

Reply:
When the reqs are in the requirements, it'll assure everyone will get them

Reply:
please remove